src/Sql/Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorCommunicator.cs

```diff
@@ -102,7 +102,7 @@ public void Remove(string resourceGroupName, string serverName)
         /// </summary>
         public Management.Sql.Models.ServerAzureADAdministrator Disable(string resourceGroupName, string serverName)
         {
-            return GetCurrentSqlClient().ServerAzureADAdministrators.DisableAzureADOnlyAuthenticationAsync(resourceGroupName, serverName).Result;
+            return null;
```

Comment:
This is temp solution just for build to pass, and it is fixed with https://github.com/Azure/azure-powershell/pull/12782 which must be checked-in after this one.